apps/prairielearn/elements/pl-big-o-input/pl-big-o-input.mustache

```diff
@@ -83,13 +83,15 @@
             data-content="{{parse_error}}"
         > Invalid <i class="fa fa-exclamation-triangle" aria-hidden="true"></i></a>
     {{/parse_error}}
-
     {{#missing_input}}
+        ${{type}}($
```

Comment:
Can you include a screenshot of what this actually renders as?

Reply:
I think it's the second screenshot here: https://github.com/PrairieLearn/PrairieLearn/pull/10433#issuecomment-2359230339

Reply:
Lol, thanks, that's what I get for jumping straight to the changed files tab 😅